tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py

```diff
@@ -335,3 +335,58 @@ def process(self, raw, identifiers, frame_meta):
                 decoded_out = decoded_out.lower()
             result.append(CharacterRecognitionPrediction(identifier, decoded_out))
         return result
+
+
+class PDPDTextRecognition(Adapter):
+    __provider__ = 'ppocr'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'vocabulary_file': PathField(description='file with decoding labels'),
+            'remove_duplicates': BoolField(
+                optional=True, default=True, description='remove duplications from result string'
+            )
+        })
+        return params
+
+    def configure(self):
+        self.labels_file = self.get_value_from_config('labels_file')
```

Comment:
Maybe the dictionary file is meant here
```suggestion
        self.labels_file = self.get_value_from_config('vocabulary_file')
```